common/src/device/provisioning/transport/ClientApiVersionHelper.cs

```diff
@@ -5,9 +5,8 @@ namespace Microsoft.Azure.Devices.Provisioning.Client
 {
     internal class ClientApiVersionHelper
     {
+        // TODO: remove - this is protocol specific.
```

Comment:
>specific [](start = 43, length = 8)

both http and amqp are using it.  mqtt should use it as well??

Reply:
You're right - I missed the one in HTTP.

Reply:
After checking - this is protocol specific to HTTP and AMQP - it should't exist for MQTT.